invokeai/backend/stable_diffusion/diffusion/custom_atttention.py

```diff
@@ -58,7 +64,6 @@ def __call__(
         is_cross_attention = encoder_hidden_states is not None
 
         # Start unmodified block from AttnProcessor2_0.
-        # vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
```

Comment:
Why remove this? This block is still unmodified. Or is this just a style preference thing?

Reply:
Ah. Must have gotten removed accidentally. Was moving around a bunch of code.